Makefile

```diff
@@ -32,24 +32,27 @@ ifeq ($(DIFF), 1)
     GIT_TREESTATE = "dirty"
 endif
 
+PROTO_DIRS = pkg/generated/protobuf/ api/proto/ protoc-builder/
+SRC = $(shell find -iname "*.go" | grep -v -e $(subst $() $(), -e ,$(strip $(PROTO_DIRS))))
+PROTO_SRC = $(shell find $(PROTO_DIRS))
+
 REKOR_LDFLAGS=-X sigs.k8s.io/release-utils/version.gitVersion=$(GIT_VERSION) \
               -X sigs.k8s.io/release-utils/version.gitCommit=$(GIT_HASH) \
               -X sigs.k8s.io/release-utils/version.gitTreeState=$(GIT_TREESTATE) \
               -X sigs.k8s.io/release-utils/version.buildDate=$(BUILD_DATE)
 SERVER_LDFLAGS=$(REKOR_LDFLAGS)
 
-GOBIN ?= $(shell go env GOPATH)/bin
-
-lint: ## Run golangci-lint checks
-	$(GOBIN)/golangci-lint run -v ./...
+GOBIN = $(abspath ./tools/bin)
 
-addlicense: ## Add licenses to source files
+lint: tools
 	$(GOBIN)/addlicense -l apache -c "The Sigstore Authors" -ignore "third_party/**" -v *
```

Comment:
to avoid GOBIN, you can also use `go tool <name>`  from https://tip.golang.org/doc/modules/managing-dependencies#tools